app/src/androidTest/java/com/github/scribeWizTeam/scribewiz/ProfilePageFragmentTest.kt

```diff
@@ -22,6 +21,20 @@ class ProfilePageFragmentTest {
 
     @Before
```

Comment:
Nicely written tests. If you have time to add additional tests, it would perhaps be useful to test more in depth the behaviour of the app when a user adds a friend that does not exist in the database, or adds themself, or the same friend twice. otherwise looks good!